src/frontend/src/sass/_table.scss

```diff
@@ -0,0 +1,44 @@
+.data-table__container {
+  padding: 0;
+
+  & > .data-table {
+    margin-bottom: 0;
+
+    & > * > tr > * {
+      padding-block: 1.2em;
+      padding-inline: 1.5em;
+    }
+
+    & > thead > tr {
+      border-bottom: 2px solid $accent-colour;
+    }
+
+    & > tbody > tr > td:nth-child(4) {
```

Comment:
I don't think `nth-child(4)` is intuitive enough to locate the status. Can we please use the className instead?